test/systemtests/util_test.go

```diff
@@ -1365,17 +1376,10 @@ func (s *systemtestSuite) SetUpTestBaremetal(c *C) {
 			time.Sleep(500 * time.Millisecond)
 		}
 	}
-	if s.fwdMode == "routing" {
-		c.Assert(s.cli.GlobalPost(&client.Global{FwdMode: "routing",
-			Name:             "global",
-			NetworkInfraType: "default",
-			Vlans:            "1-4094",
-			Vxlans:           "1-10000",
-			ArpMode:          "proxy",
-			PvtSubnet:        "172.19.0.0/16",
-		}), IsNil)
-		time.Sleep(40 * time.Second)
-	}
+
+	s.setGlobalSettings(c, s.fwdMode)
+	time.Sleep(40 * time.Second)
```

Comment:
why 40 seconds?

Reply:
it was set to 40 sec on baremetal testing and 120 sec on vagrant case, this magic number comes from previous testing, i think